packages/gluestack-ui/src/r4b/data-display/fhir-formatter.tsx

```diff
@@ -0,0 +1,15 @@
+import { FhirFormatterRendererProps } from "@bonfhir/react/r4b";
+import { Text } from "@gluestack-ui/themed";
+import { ComponentProps, ReactElement } from "react";
+
+type TextProps = ComponentProps<typeof Text>;
+
+export function GlueStackFhirFormatter(
+  props: FhirFormatterRendererProps<GlueStackFhirFormatter>,
+): ReactElement | null {
+  return <Text {...props.rendererProps?.text}>{props.formattedValue}</Text>;
```

Comment:
You cannot render a string outside of a text component React Native, so I had to replace the fragment with a Text component

Reply:
That's OK - the RN `<Text />` supports nesting so you can compose easily IMHO.